.travis.yml

```diff
@@ -78,7 +87,12 @@ matrix:
       env:
         - MB_PYTHON_VERSION=3.6
         - NP_BUILD_DEP="1.9.3"
-
+    - os: osx
+      language: generic
+      env:
+        - MB_PYTHON_VERSION=3.7
+        - NP_BUILD_DEP=1.14.5
+        - NP_TEST_DEP=1.14.5
```

Comment:
this repo doesn't use ``NP_TEST_DEP`` -- maybe remove this?

Reply:
Thanks. Fixed.